.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
What is this file? If it's temporary make clean ought to remove it.

Reply:
This is a folder that IDE creates locally. This line could be safely removed.

Reply:
What is IDE? Integrated development environment? If so which one?

Reply:
Yes, it's IntelliJ IDEA. I could move this line to my global `.gitignore` to keep this repo clean.